src/UpgradeAssistant.Mappings/mappings/AlohaKit.Animations/alohakit.animations.apimap.json

```diff
@@ -0,0 +1,7 @@
+{
+  "Xamanimation": {
```

Comment:
Should this all be in a `jsuarezruiz` folder under `mappings` to follow our naming convention guidelines?

Reply:
Arguably, yes.